tests/ccgbugs/twrong_setconstr.nim

```diff
@@ -113,7 +113,7 @@ type
     warnProveInit, warnProveField, warnProveIndex, warnGcUnsafe, warnGcUnsafe2,
     warnUninit, warnGcMem, warnDestructor, warnLockLevel, warnResultShadowed,
     warnUser,
-    hintSuccess, hintSuccessX,
+    hintSuccess, hintSuccessX, hintBuildMode,
```

Comment:
ditto here in case you need to make another change to this PR, remove this diff, otherwise keep it and it'll get removed with https://github.com/nim-lang/Nim/pull/17876